@ecomplus/widget-offers-notification/src/index.js

```diff
@@ -0,0 +1,31 @@
+
+export default (options = {}) => {
+  const { body } = window._context
```

Comment:
`window._context` também é interna, use `window.storefront.context`:

```suggestion
  const { storefront } = window
  const body = storefront && storefront.context && storefront.context.body
```